docker/app/test_small_training.py

```
@@ -0,0 +1,159 @@
+    logits = model(inputs['input_ids'], inputs['attention_mask'])
+    
+# マスク位置を特定
+mask_pos = (inputs['input_ids'] == tokenizer.mask_token_id).nonzero(as_tuple=True)[1]
+
+if len(mask_pos) > 0:
+    pos = mask_pos[0].item()
+    probs = torch.softmax(logits[0, pos], dim=-1)
+    top_k = torch.topk(probs, k=5)
+    
+    print(f"   入力: {test_text}")
+    print(f"   予測:")
```

Ruff (0.15.11)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)